### PR TITLE
OCS Alerts: Alerts for OCS plus files to reuse(helpers)

### PR DESCRIPTION
**Pull request by jmolmo**
Alerts: CephClusterErrorState, CephClusterWarningState, CephOSDVersionMismatch,
    PersistentVolumeUsageCritical, CephClusterCriticallyFull, CephClusterNearFull,
    CephClusterReadOnly, CephMonVersionMismatch, CephPoolQuotaBytesCriticallyExhausted,
    CephPoolQuotaBytesNearExhaustion

### CODE REVIEW

**Comment by jmolmo** [comment]
@iamniting , @nbalacha, Please provide review when you have time. Thanks in advance.

**Comment by iamniting** [comment]
> @iamniting , @nbalacha, Please provide review when you have time. Thanks in advance.

@jmolmo I'm uncertain as to why I received the review request from the bot, as I don't possess the expertise necessary to evaluate this PR. I believe Umanga might be better suited for the task.

**Comment by jmolmo** [comment]
@iamniting , @nbalacha , I do not know why the bot include you as reviewers.  

 @jan--f , @NautiluX, @ravitri , @RiRa12621 

Can you help me to review the PR?  It is needed a reviewer from OCS?

**Comment by RiRa12621** [comment]
Bot picks a random person from the list https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/OWNERS

**Comment by jmolmo** [comment]
@RiRa12621 I need a review on this pull request. Who can help me?

**Comment by RiRa12621** [comment]
```
  - jarrpa
  - obnoxxx
  - umangachapagain
  - agarwal-mudit
  - nbalacha
  - BlaineEXE
  - nb-ohad
  - iamniting
  ```
  
  or probabaly @NautiluX can help you too

**Comment by jmolmo** [comment]
@umangachapagain can you review the PR?

File: alerts/openshift-container-storage-operator/helpers/gatherLogs.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Optional log gathering
+
+Document Ceph Cluster health check:
+
+```bash
+    oc adm must-gather
```
Comment (umangachapagain) [inline alerts/openshift-container-storage-operator/helpers/gatherLogs.md:6]:
```suggestion
    oc adm must-gather --image=<ODF-MUST-GATHER-IMAGE>
```

**Review by umangachapagain — CHANGES_REQUESTED** [review]

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

**Pull request merged**